### ISSUE TITLE
[2.3.2] ionic generate page error --skip-import

### ISSUE THREAD

**Issue by indraraj26** [ionic-team/angular-toolkit#273]
```
ionic generate page contact
> ng.cmd generate page contact --project=app
Could not find an NgModule. Use the skip-import option to skip importing in NgModule.
[ERROR] Could not generate page.

```

This is informed you same command is working with 2.3.0

```
ionic info

Ionic:

   Ionic CLI                     : 6.11.0 (C:\Users\indra26\AppData\Roaming\npm\node_modules\@ionic\cli)
   Ionic Framework               : @ionic/angular 5.3.1
   @angular-devkit/build-angular : 0.1000.6
   @angular-devkit/schematics    : 10.0.6
   @angular/cli                  : 10.0.6
   @ionic/angular-toolkit        : 2.3.2

Capacitor:

   Capacitor CLI   : 2.4.0
   @capacitor/core : 2.4.0

Utility:

   cordova-res : 0.15.1
   native-run  : 1.0.0

System:

   NodeJS : v10.16.3 (C:\Program Files\nodejs\node.exe)
   npm    : 6.9.0
   OS     : Windows 10


```

**Comment by Jagruti-Abhyasi** [comment]
PS D:\Ionic_Project\TestApp> ionic g page ActivityDetail
Could not find an NgModule. Use the skip-import option to skip importing in NgModule.
[ERROR] Could not generate page.

Having a problem in creating new page .

**Comment by indraraj26** [comment]
for now just rollback to 2.3.0 to resolve
`npm i @ionic/angular-toolkit@2.3.0 --save-dev`

**Comment by mhartington** [comment]
Sorry about that. Should be fixed in 2.3.3
